package.json

```diff
@@ -4,13 +4,21 @@
   "description": "A book search and recommendation plan creator web app",
   "scripts": {
     "start": "parcel index.html",
-    "build": "parcel build index.html --dist-dir ./dist"
+    "build": "parcel build index.html --dist-dir ./dist",
+    "prepare": "husky"
   },
   "author": "Pranav Patani",
   "license": "ISC",
   "devDependencies": {
+    "@commitlint/cli": "^19.3.0",
```

Comment:
1. is commit lint running on every commit?
1. All the files that are present in the staging area when committing - are we running eslint and prettier on them to check for errors? - Commit should fail in case of any formatting or eslint error too. Can check this package - https://github.com/lint-staged/lint-staged

Reply:
Fixed, [commit reference](https://github.com/Pranav-Patani/BookWise/pull/4/commits/7033b4cda6823c2c4d9c68bd53d1c7b2fba95d13)

Reply:
Fixed https://github.com/Pranav-Patani/BookWise/pull/4/commits/40f1e2f1f54a7d2fb534decc769a0bc7cb001e51